docs/options.md

```diff
@@ -440,7 +440,7 @@ _First available in v2.1.0_
 
 Control whether Prettier formats quoted code embedded in the file.
 
-When Prettier identifies cases where it looks like you've placed some code it knows how to format within a string in another file, like in a tagged template in JavaScript with a tag named `html` or in code blocks in Markdown, it will by default try to format that code.
+When Prettier identifies cases where it looks like you've placed some code it knows how to format within a string in another file, like in a tagged template in JavaScript with a [tag named `html`](https://github.com/fregante/code-tag#readme) or in code blocks in Markdown, it will by default try to format that code.
```

Comment:
Don't think it's a good idea to link this directly to `code-tag`.  Maybe add a list?

Reply:
> so I avoided adding examples because the sentence is already quite long.

> extending the helper’s own readme to document this obscure platform feature

My intention is to link to other in the readme as well